pinpoint-hbase/hbase-site-standalone.xml

```diff
@@ -0,0 +1,42 @@
+<configuration>
+    <property>
+        <name>hbase.rootdir</name>
+        <value>file:///home/pinpoint/hbase</value>
+    </property>
+    <property>
+        <name>hbase.cluster.distributed</name>
+        <value>true</value>
+    </property>
+     <property>
+        <name>zookeeper.znode.parent</name>
+        <value>/hbase</value>
+    </property>
+    <property>
+        <name>hbase.zookeeper.quorum</name>
+        <value>zoo1,zoo2,zoo3</value>
+    </property>
```

Comment:
if it's stand-alone mode. Is this even necessary? At least, zoo1 is not the zookeeper hbase needs to connect. It should be the internal one.

Reply:
It would be obsolete, I just want to use the same yaml file in pinpoint-zookeeper directory.

Reply:
I think you are misunderstading the architecture of stand-alone hbase.